addons/csw/functions/fnc_initVehicle.sqf

```diff
@@ -23,9 +23,14 @@ if (!simulationEnabled _vehicle) exitWith {
 
 private _typeOf = typeOf _vehicle;
 private _configOf = configOf _vehicle;
-private _configEnabled = (getNumber (_configOf >> QUOTE(ADDON) >> "enabled")) == 1;
-private _assemblyConfig = _configEnabled && {(getText (_configOf >> QUOTE(ADDON) >> "disassembleWeapon")) != ""};
-TRACE_4("initVehicle",_vehicle,_typeOf,_configEnabled,_assemblyConfig);
+private _configCsw = _configOf >> QUOTE(ADDON);
+private _configEnabled = (getNumber (_configCsw >> "enabled")) == 1;
+private _assemblyConfig = _configEnabled && {(getText (_configCsw >> "disassembleWeapon")) != ""};
+private _hasAutofireEnabled = (getNumber (_configCsw >> "allowFireOnLoad")) == 1;
+private _forceAutofireEnabled = (getNumber (_configCsw >> "allowFireOnLoad")) == 2;
```

Comment:
```suggestion
private _configEnabled = (getNumber (_configCsw >> "enabled")) isEqualTo 1;
private _assemblyConfig = _configEnabled && {(getText (_configCsw >> "disassembleWeapon")) isNotEqualTo ""};
private _hasAutofireEnabled = (getNumber (_configCsw >> "allowFireOnLoad")) isEqualTo 1;
private _forceAutofireEnabled = (getNumber (_configCsw >> "allowFireOnLoad")) isEqualTo 2;
```

Reply:
same as above, i think this is slower via profiling in the ace discord